nvflare/lighter/impl/template.py

```diff
@@ -20,6 +20,10 @@
 
 
 class TemplateBuilder(Builder):
+    """Loads the content of the template_file and the authz_def (section of template file with fixed authorization
+    definitions) into two key-value pairs in the build context.
+    """
+
```

Comment:
Why do we need this class though?

For me this is more like a loader rather than "builder".

Reply:
This is load the template file.  It's possible other developers have different template files and need to load them differently.  Also, because it only implements the initialize method, it's not a full 'builder' per se.